### PR TITLE
Translate front-end and settings. Fix error.

### PR DESCRIPTION
**Pull request by ne-ray**

### CODE REVIEW

File: models/reservation/columns.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ columns:
         label: vojtasvoboda.reservations::lang.reservation.returning
         type: text
         invisible: true
-        sortable: true
+        sortable: false
```
Comment (vojtasvoboda) [inline models/reservation/columns.yaml:41]: Why this?

Comment (ne-ray) [reply models/reservation/columns.yaml:41]: SQL Exception `column returning not exists`

**Review by vojtasvoboda — APPROVED** [review]

**Comment by vojtasvoboda** [comment]
I've tested all translations, update Czech lang file and tested fix with loading reserved dates. Thanks for fixing this bug with reservation date.

File: models/reservation/columns.yaml (hunk 1 above)
Comment (vojtasvoboda) [reply models/reservation/columns.yaml:41]: Ok. This is definitely bug. I marked this bug at Readme file.

**Pull request merged**